### PR TITLE
[Core] Move repository method from order to core

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | yes |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
The upgrade note is wrong and you should explain why it does not work in this PR.

**Comment by lchrusciel** [comment]
@pjedrzejewski done

File: src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php [hunk 1]
```diff
@@ -209,4 +211,23 @@ public function findLatestInChannel($count, ChannelInterface $channel)
...
+            ->andWhere('o.paymentState != :paymentState')
+            ->andWhere('o.state = :orderState')
+            ->andWhere('o.checkoutCompletedAt < :terminalDate')
+            ->setParameter('checkoutState', OrderCheckoutStates::STATE_COMPLETED)
+            ->setParameter('paymentState', OrderPaymentStates::STATE_PAID)
+            ->setParameter('orderState', OrderInterface::STATE_NEW)
+            ->setParameter('terminalDate', $terminalDate)
+            ->getQuery()
+            ->getResult()
+            ;
```
Comment (Zales0123) [inline src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php]: Wrong indent.

File: UPGRADE.md [hunk 2]
```diff
@@ -86,6 +86,9 @@ After:
 
 ### Order / OrderBundle
 
+* Method `findOrdersUnpaidSince` from `Sylius\Bundle\OrderBundle\Doctrine\ORM\OrderRepository` has been moved to `Sylius\Bundle\CoreBundle\Doctrine\ORM\OrderRepository` as it has been depended on Core component.
```
Comment (NeverResponse) [inline UPGRADE.md]: `from` => `of`
`as it has been depended` (wot) => `as it depends`

File: UPGRADE.md [hunk 3]
```diff
@@ -86,6 +86,9 @@ After:
 
 ### Order / OrderBundle
 
+* Method `findOrdersUnpaidSince` from `Sylius\Bundle\OrderBundle\Doctrine\ORM\OrderRepository` has been moved to `Sylius\Bundle\CoreBundle\Doctrine\ORM\OrderRepository` as it has been depended on Core component. 
+ If you have used `findOrdersUnpaidSince` method you had to require CoreBundle anyway. If you haven't used this method, but you have been force to require `CoreBundle` than you can remove this dependency.
```
Comment (NeverResponse) [inline UPGRADE.md]: `If you haven't been using this method, you can remove the 'CoreBundle' dependency.`
instead of those two, wdyt?

**Comment by pjedrzejewski** [comment]
👍 👍 👍

**Pull request merged**